Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx

```diff
@@ -10,10 +10,43 @@
 
 =========================================================================auto=*/
 
+#include "vtkMRMLBSplineTransformNode.h"
 #include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLScene.h"
 #include "vtkMRMLTransformNode.h"
 
 #include <vtkGeneralTransform.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkTransform.h>
+
+bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
```

Comment:
> collect all the utilities we find useful and put them in
vtkAddon
:+1: 

> but I would hesitate to do that in this commit

Good point. This can definitively be done in an other commit.

Simply adding a note in this commit message would be good.